apps/web/app/api/v1/management/surveys/route.ts

```diff
@@ -0,0 +1,52 @@
+import { responses } from "@/lib/api/response";
+import { authenticateRequest } from "@/app/api/v1/auth";
+import { NextResponse } from "next/server";
+import { transformErrorToDetails } from "@/lib/api/validator";
+import { createSurvey, getSurveys } from "@formbricks/lib/services/survey";
+import { ZSurveyInput } from "@formbricks/types/v1/surveys";
+import { DatabaseError } from "@formbricks/types/v1/errors";
+
+export async function GET(request: Request) {
+  try {
+    const authentication = await authenticateRequest(request);
+    console.log(authentication);
+    if (!authentication) {
+      responses.notAuthenticatedResponse();
+    }
+    const surveys = await getSurveys(authentication.environmentId!);
+    return responses.successResponse(surveys);
+  } catch (error) {
+    if (error instanceof DatabaseError) {
+      return responses.badRequestResponse(error.message);
+    }
+    throw error;
+  }
+}
+
+export async function POST(request: Request): Promise<NextResponse> {
```

Comment:
Please use `PUT` for a update

Reply:
I think this route is used for survey creating and update survey is already present in `surveys/[surveyId]/routes.ts`. 🤔